src/tools/cephfs/cephfs-shell

```diff
@@ -733,6 +733,11 @@ exists.')
                     self.poutput(path, end=':\n')
                 items = sorted(ls(path),
                                key=lambda item: item.d_name)
+            if items == []:
+                self.perror('{}: no such directory exists'.format(path),
+                            end='\n', apply_style=True)
+                continue
```

Comment:
and then this block should not need this special handling at all, right?

Reply:
This should be fine. I have updated the patch.